src/contract.ts

```diff
@@ -244,7 +380,7 @@ export class Contract {
    * @param inputs  - list of inputs(fields) that are in the abi
    * @return {Calldata} - parsed arguments in format that contract is expecting
    */
-  private compileCalldata(args: Args, inputs: AbiEntry[]): Calldata {
+  compileCalldata(args: Args, inputs: AbiEntry[]): Calldata {
```

Comment:
can we keep everything either private, protected or public? I think this method (and following) should be protected

Reply:
I agree that this method should be private or protected because we already have a `compileCalldata` function exported from `utils/stark`